server/src/db/seeders/ReviewSeeder.js

```diff
@@ -0,0 +1,43 @@
+import { Review } from "../../models/index.js";
+
+class ReviewSeeder {
+  static async seed() {
+    const reviewsData = [
+      {
+      userId: 1,
+      shoeId: 1,
```

Comment:
this hardcoding of id values will likely cause problems if a record with that value does not exist. Instead we should query for the related data we are looking for here and reference the actual objects `id`